spec/dummy/config/routes.rb

```diff
@@ -9,6 +9,8 @@
     namespace :v1x0, :path => "v1.0" do
       get "/openapi.json", :to => "root#openapi"
       post "graphql" => "graphql#query"
+      get '/error', :to => "errors#error"
```

Comment:
Is this for test purposes rather than reusing an existing controller?

Reply:
Yes - I can move the routes to another controller if you would prefer that.